src/style.css

```diff
@@ -1 +0,0 @@
-/* add styles here */
```

Comment:
Styles should be here in one file.

Reply:
Good morning! One of checklist point was "Each BEM block has its own separate file". Should I move styles from rest.css to style.css ? If yes, where import BEM block files?

Reply:
@FaiHamid I wrote a clarification to your question in response to the QnA chat